server/commands/check.go

```diff
@@ -254,11 +254,28 @@ func (query *CheckQuery) resolveDirectUserSet(
 		}
 
 		// If a single star is available, then assume user exists and break.
```

Comment:
Please remove this comment, the code is clear enough

Reply:
Done